### PR TITLE
feat(console): Support of existing secret

### PR DESCRIPTION
**Pull request by AurelieMarcuzzo**

### CODE REVIEW

File: charts/console/templates/tests/02-credentials-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: conduktor-platform
+type: Opaque
+stringData:
+  CDK_ORGANIZATION_NAME: {{ .Values.config.organization.name | quote }}
+  CDK_ADMIN_EMAIL: {{ .Values.config.admin.email | quote }}
+  CDK_ADMIN_PASSWORD: {{ .Values.config.admin.password | quote }}
+  CDK_DATABASE_PASSWORD: {{ .Values.config.database.password | quote }}
+  CDK_DATABASE_USERNAME: {{ .Values.config.database.username | quote }}
```
Comment (qboileau) [inline charts/console/templates/tests/02-credentials-secret.yaml:16]: Missing base64 encode for all secrets, you can replace `quote` with `b64enc | quote`

File: charts/console/templates/tests/02-credentials-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{{- if and (eq .Values.test true) .Values.config.existingSecret }}
+{{- $secretName := include "conduktor.platform.secretName" . }}
```
Comment (qboileau) [inline charts/console/templates/tests/02-credentials-secret.yaml:2]: no need for a variable if only used once

File: .github/workflows/chart-test.yaml [hunk 3]
```diff
@@ -31,58 +31,59 @@ jobs:
...
-        run: make k3d-up
-
-      - name: Run chart-testing (install)
-        run: ct install --config .github/ct-config.yaml
-        if: steps.list-changed.outputs.changed == 'true'
-
-      - name: Create k3d cluster
-        if: always()
-        run: make k3d-down
+# Disabled for now as we don't have enough resources to run it
```
[the remark this answers is not in the document]
Comment (AlexandreBrg) [reply .github/workflows/chart-test.yaml:34]: We should be using the lint in addition to test, that's why I disabled the job manualy

File: charts/console/templates/tests/02-credentials-secret.yaml (hunk 1 above)
Comment (AlexandreBrg) [reply charts/console/templates/tests/02-credentials-secret.yaml:16]: As we're using stringData, no need for b64

Comment (qboileau) [reply charts/console/templates/tests/02-credentials-secret.yaml:16]: I missed that thanks

File: charts/console/templates/platform/deployment.yaml [hunk 4]
```diff
@@ -69,7 +69,6 @@ spec:
       securityContext: {{- include "common.tplvalues.render" (dict "value" .Values.platform.containerSecurityContext "context" $) | nindent 10 }}
       {{- else }}
       securityContext:
-        readOnlyRootFilesystem: true
```
Comment (qboileau) [inline charts/console/templates/platform/deployment.yaml:72]: ok to leave on container ?

**Review by qboileau — APPROVED** [review]

File: charts/console/templates/platform/deployment.yaml (hunk 4 above)
Comment (AlexandreBrg) [reply charts/console/templates/platform/deployment.yaml:72]: It is indeed, I believe it's not only on podSecurityContext for some distribution but on all for container

**Pull request merged**